src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx

```diff
@@ -105,6 +106,7 @@ export default class AddWebLink extends React.Component {
                   </label>
                   <div styleName={webLinkValid ? 'validation-bar url' : 'validation-bar url error-bar'}>
                     <input
+                      autocapitalize="off"
```

Comment:
I think the correct attribute for React is `autoCapitalize`

Reply:
@ThomasKranitsas fixed in latest commit